operacje_na_listach/Function_to_reject.py

```diff
@@ -0,0 +1,7 @@
+def reject(list):
+    even=[]
+    for i in list:
+        if i%2== 0:# jeśli reszta z dzielenia =0, liczby parzyste
```

Comment:
please extract it to separate function named `is_divisible` that takes 2 arguments - a number and divisor. So the output of `is_divisible(10, 2)` would be `True`